src/components/App.js

```diff
@@ -1,17 +1,112 @@
-import React from 'react';
+import React from "react";
+import Films from "./Films";
+import Search from "./Search";
 
 class App extends React.Component {
-  constructor(){
+  constructor() {
     super();
+
+    this.state = { films: [], pageNum: 1, currentSearchFlim: "love", unFound: {gif: '', h6: ''}, hideButtons: true};
```

Comment:
It would be better to break out each property definition onto own line to make code easier to read